app/models/source.rb

```diff
@@ -24,4 +24,8 @@ def default_endpoint
     default = endpoints.detect(&:default)
     default || endpoints.build(:default => true, :tenant => tenant)
   end
+
+  def super_key
+    authentications.find_by!(:authtype => "superkey")
```

Comment:
Shouldn’t this be something like `find auth with a type thas has is_superkey set to true`?

Reply:
I could see that being necessary if there was a use-case of having multiple superkeys on a source - but basically this is assuming only one SuperKey authentication will be created off of the source with `authtype` set to `superkey`, which should be the case. It will error out if there isn't a superkey authentication under the source. 

If we do want to allow multiple superkey sets of credentials per source we can revisit it, but I don't see a use-case for it. 

----

If we don't want to use `authtype => superkey` (which i didn't think of as an option until now) then I'd have to do a more sophisticated lookup, I can change it if we still want to use `authtype => access_key_secret_key`.

Reply:
I thought we had made this decision in this PR: https://github.com/RedHatInsights/sources-api/pull/310/files#diff-e77d362e01f7845504382af8c7c60f0bbacfab1fbc642e30c2d776928bd942e8R7

Reply:
Updating in #324